packages/sdk/src/reads/getIsExternalPositionsValuesZero.ts

```diff
@@ -0,0 +1,33 @@
+import { publicClientMainnet } from "../../tests/globals.js";
+import { getActiveExternalPositions } from "./getActiveExternalPositions.js";
+import { getExternalPositionAssets } from "./getExternalPositionAssets.js";
+import type { Address } from "viem";
+
+export async function getIsExternalPositionsValueZero(vaultProxy: Address) {
+  const addresses = await getActiveExternalPositions(publicClientMainnet, {
+    vaultProxy,
+  });
+
+  const values = await Promise.all(
+    addresses.map(async (address) => {
+      const { debtAssets, managedAssets } = await getExternalPositionAssets(publicClientMainnet, {
+        externalPosition: address,
+      });
+
+      const debtAssetsValue =
+        debtAssets.map((asset) => asset.amount).reduce((acc, amount) => (acc ?? 0n) + (amount ?? 0n), 0n) ?? 0n;
```

Comment:
there is bug in the getManagedAssets, and getDebtAssets. It should return `{asset: Address, amount: bigint}[]`, assert in those functions for undefined, and throw an error if undefined
![Screenshot 2023-09-12 at 10 36 48](https://github.com/enzymefinance/sdk/assets/67273035/7e93d3a1-cae4-43ec-90c9-71238e09d4d2)

Reply:
I assert that `assets` and `amounts` arrays have the same length, and then if the `amount` is `undefined` I replace it with `0n`, as it seems possible that no amount might return `undefined` instead of `0n`, and it would be wrong to throw an error in this instance.